src/org/opensolaris/opengrok/configuration/messages/Message.java

```diff
@@ -339,29 +339,32 @@ public Builder(final Class<T> cl) {
 
         public T build() {
             T result = createMessage(messageClass);
+            fillData(result);
```

Comment:
I wonder if now it works without the cast why it didn't before when you had it inline

Reply:
Now it works without cast because `T` is covariant type to `Message`. However, before was the problem that Java somehow could not deduce that the modifications to `private` members were done inside the enclosing class which has the private members. I don't know the reason why :D

Reply:
ok :D seems fun